terraso_backend/apps/audit_logs/services.py

```diff
@@ -19,32 +20,30 @@ class _AuditLogService:
 
     def log(
         self,
-        user: object,
-        action: int,
+        user: User,
+        action: api.ACTIONS,
         resource: object,
-        metadata: typing.Optional[List[api.KeyValue]] = None,
+        metadata: typing.Optional[api.KeyValue] = None,
```

Comment:
is this a more accurate type?
```suggestion
        metadata: typing.Optional[dict[str, any]] = None,
```